package.json

```diff
@@ -1,32 +1,50 @@
 {
-  "name": "vscode-teddy-snippets",
-  "displayName": "Teddy Snippets",
-  "description": "Snippets for the Teddy templating language",
-  "version": "1.0.1",
+  "name": "vscode-teddy",
+  "displayName": "Teddy",
+  "description": "Support for the Teddy templating language in VS Code",
+  "author": "Roosevelt Framework Team <rooseveltframework@gmail.com>",
+  "contributors": [
+    {
+      "name": "Contributors",
+      "url": "https://github.com/rooseveltframework/vscode-teddy/graphs/contributors"
+    }
+  ],
+  "version": "1.0.2",
   "publisher": "lannonbr",
```

Comment:
The `publisher` should be updated to now reference the new publisher group name instead of just my personal one